src/client/elm/Explorer/View.elm

```diff
@@ -1856,56 +1889,89 @@ viewEventUpdateEnqueuedDetails ctx event =
             paragraph [] [ text <| "Update the minimum staked amount for becoming a baker to " ++ T.amountToString threshold ]
 
         AddAnonymityRevokerPayload (ArInfo anonymityRevokerInfo) ->
-            paragraph [] <| text ("Add a new anonymity revoker. ") :: displayArIp anonymityRevokerInfo
+            paragraph [] <| text "Add a new anonymity revoker. " :: displayArIp anonymityRevokerInfo
 
         AddIdentityProviderPayload (IpInfo identityProviderInfo) ->
-            paragraph [] <| text ("Add a new identity provider. ") :: displayArIp identityProviderInfo
+            paragraph [] <| text "Add a new identity provider. " :: displayArIp identityProviderInfo
 
 
 displayArIp : ArIpInfo -> List (Element Msg)
 displayArIp info =
-    let descr = info.description
-    in displayName descr.name
-    :: displayIdentity info.identity
-    :: displayDescription descr.description
-    :: displayWebsite descr.url
+    let
+        descr =
+            info.description
+    in
+    displayName descr.name
+        :: displayIdentity info.identity
+        :: displayDescription descr.description
+        :: displayWebsite descr.url
 
 
-displayName: String -> Element Msg
-displayName = displayStr "Name"
+displayName : String -> Element Msg
+displayName =
+    displayStr "Name"
 
 
-displayIdentity: Identity -> Element Msg
-displayIdentity id = let i = case id of
-                             ArIdentity ar -> ar
-                             IpIdentity ip -> ip
-                     in displayStr "Identity" <| String.fromInt i
+displayIdentity : Identity -> Element Msg
+displayIdentity id =
+    let
+        i =
+            case id of
+                ArIdentity ar ->
+                    ar
 
+                IpIdentity ip ->
+                    ip
+    in
+    displayStr "Identity" <| String.fromInt i
+
+
+displayDescription : String -> Element Msg
+displayDescription =
+    displayStr "Description"
 
-displayDescription: String -> Element Msg
-displayDescription = displayStr "Description"
 
 
 {- Format a nonempty string with its corresponding attribute. This is used to display information on update
    transactions.
 -}
-displayStr: String -> String -> Element Msg
-displayStr attrName str = let elem s = text <| attrName ++ ": " ++ s ++ ". "
-                          in case Regex.fromString "[\\. ]*$" of -- to remove trailing spaces and periods
-                               Nothing -> elem (String.trim str)
-                               Just regex -> let trimmed = Regex.replace regex (\_ -> "") str
-                                             in if trimmed == "" then text "" else elem trimmed
+
+
+displayStr : String -> String -> Element Msg
+displayStr attrName str =
+    let
+        elem s =
+            text <| attrName ++ ": " ++ s ++ ". "
+    in
+    case Regex.fromString "[\\. ]*$" of
+        -- to remove trailing spaces and periods
```

Comment:
It would be nice with a comment on why this is necessary as it seems a bit strange.